src/lcm/next_state.py

```diff
@@ -63,20 +71,20 @@ def get_next_state_function(
 
 def get_next_stochastic_weights_function(
     model: InternalModel,
+    next_stochastic_variables: tuple[str, ...],
```

Comment:
```suggestion
    next_stochastic_states: tuple[str, ...],
```

I think there is value in being specific here, unless it clashes with terminology used throughout.